independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/MultiInvoker.java

```diff
@@ -291,53 +291,15 @@ public void handle(Buffer buffer) {
                 try {
                     byte[] bytes = buffer.getBytes();
                     MediaType mediaType = response.getMediaType();
+                    ByteArrayInputStream in = new ByteArrayInputStream(bytes);
```

Comment:
I can see one drawback of this new approach: we are now allocating a large `ByteArrayInputStream` instead of several smaller ones.

Might or might not end up being a problem.